# This contains the motor commands for a 2 motor air vehicle. (0.0 - 1.0)
float32 m1
float32 m2
